cmds/cmd_init.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 The KCL Authors. All rights reserved.
+
+package command
+
+import (
+	"os"
+
+	"github.com/urfave/cli/v2"
+	conf "kusionstack.io/kpm/core/conf"
+	reporter "kusionstack.io/kpm/core/reporter"
+	ops "kusionstack.io/kpm/ops"
+)
+
+// NewInitCmd new a Command for `kpm init`.
+func NewInitCmd() *cli.Command {
+	return &cli.Command{
+		Hidden: false,
+		Name:   "init",
+		Usage:  "initialize new module in current directory",
+		Action: func(c *cli.Context) error {
+			if c.NArg() == 0 {
+				reporter.Report("kpm: module name must be specified.")
+				reporter.ExitWithReport("kpm: run 'kpm init help' for more information.")
+			}
+			modName := c.Args().First()
+			pwd, err := os.Getwd()
+
+			if err != nil {
+				reporter.Fatal("kpm: internal bugs, please contact us to fix it")
+			}
+
+			config := conf.NewEmptyConf().SetName(modName).SetExecPath(pwd)
```

Comment:
Why set pwd to execpath? Remove it?

For cli options, u can ref https://github.com/KusionStack/kusion/tree/main/pkg/cmd/init

Reply:
fixed.